Source/Core/DolphinQt/Debugger/RegisterWidget.cpp

```diff
@@ -101,6 +101,7 @@ void RegisterWidget::ConnectWidgets()
   connect(m_table, &QTableWidget::customContextMenuRequested, this,
           &RegisterWidget::ShowContextMenu);
   connect(m_table, &QTableWidget::itemChanged, this, &RegisterWidget::OnItemChanged);
+  connect(&Settings::Instance(), &Settings::DebugFontChanged, m_table, &QTableWidget::setFont);
```

Comment:
```suggestion
  connect(&Settings::Instance(), &Settings::DebugFontChanged, m_table, &QWidget::setFont);
```
so that this doesn't have to be changed in case we replace QTableWidget with something else in the future, and because setFont is in QWidget